### PR TITLE
Simplify report submitting

### PR DESCRIPTION
**Pull request by benrudolph**
@emord @biyeun this simplifies the report submit button. I think there were race conditions and perhaps the upgrade in jquery brought them to light. In any case this works locally for all the major reports
http://manage.dimagi.com/default.asp?237437
cc: @gcapalbo

### CODE REVIEW

File: corehq/apps/reports/static/reports/javascripts/reports.async.js [hunk 1]
```diff
@@ -160,7 +136,10 @@ var HQAsyncReport = function (o) {
                     self.hqLoading.fadeIn();
                 }
 
-            }
+            },
+            complete: function() {
+                self.standardReport.filterSubmitButton.button('reset');
```
Comment (benrudolph) [inline corehq/apps/reports/static/reports/javascripts/reports.async.js]: this is needed for an async report to re-enable the apply button. there's some complicated logic above, but i don't think it's necessary. `reset` keeps all the styles from what i noticed (perhaps fixed in some version upgrade)

**Review by emord — APPROVED** [review]

**Comment by biyeun** [comment]
@benrudolph looks good but there are merge conflicts

**Comment by benrudolph** [comment]
updated, but think it makes sense to wait til monday to merge. i still like the changes that this PR makes to the apply button

**Comment by gcapalbo** [comment]
@benrudolph is this good to merge now?

**Comment by benrudolph** [comment]
i'm gonna punt on this for now. i don't have bandwidth to ensure this is correct for all the reports we have since the logic that was happening was a bit magical to me. it makes me nervous that there are a lot of reports that reuse this and it's hard to tell what exactly this did for them

**Pull request closed without merging**